templates/python/custom-copilot-basic/appPackage/manifest.json.tpl

```diff
@@ -31,7 +31,20 @@
                 "groupChat"
             ],
             "supportsFiles": false,
-            "isNotificationOnly": false
+            "isNotificationOnly": false,
+            "commandLists": [
+                {
+                    "scopes": [
+                        "personal"
+                    ],
+                    "commands": [
+                        {
+                            "title": "Hi",
```

Comment:
Can you update the content based on Junjie's feature design loop?

Reply:
I checked the loop. The example picture shows something like an assistant agent. Those prompt are not for a basic chat bot.
For details of prompts, need @MuyangAmigo 's suggestions.

Reply:
It's in the project template requirement section. Copied here for this template:
Prompt 1: Hello, how can you help me?
Prompt 2: How can I develop apps with Teams Toolkit?

Reply:
> It's in the project template requirement section. Copied here for this template: Prompt 1: Hello, how can you help me? Prompt 2: How can I develop apps with Teams Toolkit?

Thanks! There are some issues those exceeded maximum characters, please check with Qinghui, I have asked him to truncate and please align across different templates, @frankqianms

Reply:
Thank you @blackchoey @MuyangAmigo ! Will check it.